lib/logitech_receiver/descriptors.py

```diff
@@ -232,11 +232,12 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 							_FS.lowres_smooth_scroll(),
 							_FS.pointer_speed(),
 				])
-_D('Wireless Mouse M185/M235', codename='M185/M235', protocol=4.5, wpid='4055',
-				settings=[
-							_FS.lowres_smooth_scroll(),
-							_FS.pointer_speed(),
-				])
+# wpid 4055 is the new M310
+#_D('Wireless Mouse M185/M235', codename='M185/M235', protocol=4.5, wpid='4055',
```

Comment:
Why just not put `/M310`.

Reply:
If 4055 can really be three different "bodies" then I guess that is correct.  OK, I'll do that.  It will be a smaller change.

Reply:
Yes, I believe so. Probably different shells with the same internals.

Reply:
OK, I made a clean change to just add M310 for wpid 4055 (and a put in a comment to show what is going on)